dbclient/metrics-jdbc/src/main/java/io/helidon/dbclient/metrics/jdbc/JdbcMetricsTimer.java

```diff
@@ -79,13 +79,11 @@ public Snapshot getSnapshot() {
 
     @Override
     public void update(Duration duration) {
-        // TODO 3.0.0-JAKARTA
         meter.update(duration);
     }
 
     @Override
     public Duration getElapsedTime() {
-        // TODO 3.0.0-JAKARTA
         return Duration.ZERO;
```

Comment:
Is there no way to solve this? Returning zero as an elapsed time seems wrong

Reply:
@tjquinno, any comment on this?

Reply:
I pushed two files to Keith's branch to address this.